### PR TITLE
Add notebook frontend configuration doc

### PR DESCRIPTION
**Pull request by willingc**
Addresses #457

### CODE REVIEW

File: docs/source/frontend_config.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+.. note::
+
+    The ability to configure the notebook frontend UI and preferences is
+    still a work in progress.
+
+This document is a rough explanation on how you can persist some configuration
+options for the notebook JavaScript.
+
+There is no exhaustive list on all the configuration option are most options
```
Comment (jdfreder) [inline docs/source/frontend_config.rst]: s/list on/list of/

File: docs/source/frontend_config.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          CodeCell:{
+            cm_config:{indentUnit: null} # only change here.
+          }
+        }
+    config.update(patch)
+
+Reload the notebook in your browser and the default indent should again be two
+spaces.
+
+Persisting conguration settings
```
Comment (jdfreder) [inline docs/source/frontend_config.rst]: configuration

**Comment by willingc** [comment]
@jdfreder Thanks for the review. Good catches. :ocean: 

I will loop back and correct this evening.

**Comment by minrk** [comment]
Thanks, @willingc!

**Pull request merged**